### DISCUSSION TITLE
Is installation in `. /bin` but not `/bin` an expected behavior when use bash script?

### DISCUSSION THREAD

**Discussion by AuroraTea** [nektos/act#2441] [Q&A]
When use
```
curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/nektos/act/master/install.sh | sudo bash
```

The act was installed in `./bin` but not `/bin`

I see that the script does say to install on . /bin, but I don't understand why that's the case.

**Comment by jsoref** [comment]
The help says so:
https://github.com/nektos/act/blob/f75a2d8b38cc3b929ab652c0994d6d33f6584ed9/install.sh#L12

The code here does the work:
https://github.com/nektos/act/blob/f75a2d8b38cc3b929ab652c0994d6d33f6584ed9/install.sh#L29

You can see that GitHub's highlighter understands a bit of shell:
```sh
BINDIR=${BINDIR:-./bin} 
```
the `${var_name:-fallback}` will use `$var_name` if available or `fallback` if not.

Such questions are probably best resolved by reading shell documentation, e.g. https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html
